vespalib/src/tests/stllike/hashtable_test.cpp

```diff
@@ -73,6 +73,48 @@ TEST("require that hashtable<int> can be copied") {
     EXPECT_EQUAL(42, *table2.find(42));
 }
 
+TEST("require that you can insert duplicates") {
+    using Pair = std::pair<int, vespalib::string>;
+    using Map = hashtable<int, Pair, vespalib::hash<int>, std::equal_to<int>, First<int, Pair>>;
```

Comment:
consider aliasing First to Select1st (use actual code instead of custom code that does the same)

Reply:
Done